policykit/policyengine/views.py

```diff
@@ -438,6 +438,7 @@ def clean_up_proposals(action, executed):
             p.save()
 
     p = action.proposal
+    # what is going on here
```

Comment:
Can we remove the `clean_up_proposals` function in favor of:
* After running the `pass_policy` block, always mark the action.proposal as `PASSED`. If it is a bundle, also mark the bundle as `PASSED`.
* After running the `fail_policy` block, always mark the action.proposal as `FAILED`. If it is a bundle, also mark the bundle as `FAILED`.
?

Reply:
leaving as is, created https://github.com/amyxzhang/policykit/issues/342